sentry-core/src/scope/real.rs

```diff
@@ -178,6 +178,9 @@ impl Scope {
 
     /// Sets the user for the current scope.
```

Comment:
Can you update this doc comment.

What happens when there is a session?  When will the session not see this update?